bolt/src/Target/AArch64/AArch64MCPlusBuilder.cpp

```diff
@@ -337,41 +351,80 @@ class AArch64MCPlusBuilder : public MCPlusBuilder {
     return Expr;
   }
 
+  bool getOpNum(const MCInst &Inst,
+                unsigned &OpNum) const {
```

Comment:
It's not clear what operand number we are getting here. Based on the code, I guess this is selecting the target symbol? so maybe rename it to "getSymbolRefOperandNum?" Also, instead of using OpNum as an output parameter, we can just return an optional to make it simpler:

`  Optional<unsigned> getSymbolRefOperandNum(const MCInst &Inst);`

Reply:
Thanks! But I think we need to stay with this argument since getTargetSymbol has one